accounts/abi/encodepacked.go

```diff
@@ -0,0 +1,111 @@
+package abi
+
+import (
+	"encoding/binary"
+	"errors"
+	"fmt"
+	"math/big"
+	"reflect"
+
+	"github.com/ethereum/go-ethereum/common"
+)
+
+// EncodePacked implements the non-standard encoding available in solidity.
+// Since encoding is ambigious there is no decoding function.
+// See
+func EncodePacked(args []Type, values []interface{}) ([]byte, error) {
```

Comment:
If this is merged I would suggest to go with `SolidityEncodePacked` to highlight the fact this is some Solidity specific feature.

Reply:
> 
ethers-js impl: `ethers.utils.solidityPack`